app/src/main/java/com/globant/marvelmvvm/view/activity/MainActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.globant.marvelmvvm.view.activity
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.appcompat.app.ActionBarDrawerToggle
+import androidx.core.view.GravityCompat
+import androidx.navigation.Navigation
+import com.globant.marvelmvvm.R
+import com.google.android.material.navigation.NavigationView.OnNavigationItemSelectedListener
+import kotlinx.android.synthetic.main.activity_main.activity_main_toolbar
+import kotlinx.android.synthetic.main.activity_main.activity_main_navigation_view
+import kotlinx.android.synthetic.main.activity_main.activity_main_drawer_layout
+
+class MainActivity : AppCompatActivity(), OnNavigationItemSelectedListener {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        setSupportActionBar(activity_main_toolbar)
+
+        val toggle = ActionBarDrawerToggle(this, activity_main_drawer_layout, activity_main_toolbar, ZERO, ZERO)
+        activity_main_drawer_layout.addDrawerListener(toggle)
+        toggle.syncState()
+
+        activity_main_navigation_view.setNavigationItemSelectedListener(this)
+    }
+
+    override fun onNavigationItemSelected(item: MenuItem): Boolean {
+        val navController = Navigation.findNavController(this, R.id.activity_main_active_fragment)
+        when (item.itemId) {
+            R.id.allCharactersFragment -> {
+                activity_main_toolbar.title = getString(R.string.string_all_characters)
+                Toast.makeText(this, getString(R.string.string_all_characters_message), Toast.LENGTH_SHORT).show()
+                navController.navigate(R.id.allCharactersFragment)
+            }
+            R.id.specificCharacterFragment -> {
+                activity_main_toolbar.title = getString(R.string.string_specific_character)
+                Toast.makeText(this, getString(R.string.string_specific_character_message), Toast.LENGTH_SHORT).show()
+                navController.navigate(R.id.specificCharacterFragment)
```

Comment:
Look than in both cases repeat the methods with different s "values", so you can create a method and gives the different value in each case

Reply:
Done!